terraform/eks/main.tf

```diff
@@ -22,6 +22,10 @@ terraform {
     kubernetes = {
       version = "~> 1.13"
     }
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.7.0"
```

Comment:
we use >= for kubectl and ~> for kubernetes, is it intended? or we are okay with big version upgrade?

Reply:
This is a file ported from @pxaws 's PR. I think he could possibly give a good answer~